src/components/cards/CardBadges.vue

```diff
@@ -145,7 +147,7 @@ export default {
 			this.$store.dispatch('assignCardToUser', {
 				card: this.copiedCard,
 				assignee: {
-					userId: OC.getCurrentUser().uid,
+					userId: getCurrentUser().uid,
```

Comment:
:warning: the replacement returns `null` if there is no user and the object with UID only when there is one

Reply:
Thanks, although that should be fine for now as the whole frontend currently assumes that the user is logged in.

Reply:
Makes sense. You can also use `getCurrentUser()?.uid` to be on the safe side :)